students/tlugosan/lesson02/FizzBuzz.py

```diff
@@ -0,0 +1,22 @@
+# Print numbers from 1 to 100
+# but for multiple of 3 print Fizz
+# for multiple of 5 print BUzz
+# for multiple of both 3 and 5 print FizzBuzz
+
+def FizzBuzz():
```

Comment:
per pep8 python style guide function names should be snake_case -> `def fizz_buzz`